controllers/install/lookout_controller.go

```diff
@@ -107,32 +112,48 @@ func (r *LookoutReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	}
 
 	if components.Deployment != nil {
+		logger.Info("Upserting Lookout Deployment object")
 		if _, err := controllerutil.CreateOrUpdate(ctx, r.Client, components.Deployment, mutateFn); err != nil {
 			return ctrl.Result{}, err
 		}
 	}
 
 	if components.Service != nil {
+		logger.Info("Upserting Lookout Service object")
 		if _, err := controllerutil.CreateOrUpdate(ctx, r.Client, components.Service, mutateFn); err != nil {
 			return ctrl.Result{}, err
 		}
 	}
 
-	// now do init logic
+	if components.IngressWeb != nil {
+		logger.Info("Upserting Lookout Ingress object")
+		if _, err := controllerutil.CreateOrUpdate(ctx, r.Client, components.IngressWeb, mutateFn); err != nil {
+			return ctrl.Result{}, err
+		}
+	}
+
+	logger.Info("Successfully reconciled Lookout object", "durationMilis", time.Since(started).Milliseconds())
 
 	return ctrl.Result{}, nil
 }
 
 type LookoutComponents struct {
-	Ingress        *networking.Ingress
 	Deployment     *appsv1.Deployment
+	IngressWeb     *networking.Ingress
+	Secret         *corev1.Secret
 	Service        *corev1.Service
 	ServiceAccount *corev1.ServiceAccount
-	Secret         *corev1.Secret
-	Job            *batchv1.Job
+	// ToDo: add other components
+	// CronJob        *batchv1beta1.CronJob
+
+	Job *batchv1.Job
 }
 
 func generateLookoutInstallComponents(lookout *installv1alpha1.Lookout, scheme *runtime.Scheme) (*LookoutComponents, error) {
+	// serviceAccount := r.createServiceAccount(lookout)
```

Comment:
You should probably enable ServiceAccount at some point. Leave for a followup or w/e.  But will call it out.